UUID::Tiny is only marked to support amd64 on Gentoo. Use Data::UUID instead



--- a/slimserver.pl
+++ b/slimserver.pl
@@ -475,8 +475,9 @@
 
 	# Generate a UUID for this SC instance on first-run
 	if ( !$prefs->get('server_uuid') ) {
-		require UUID::Tiny;
-		$prefs->set( server_uuid => UUID::Tiny::create_UUID_as_string( UUID::Tiny::UUID_V4() ) );
+		require Data::UUID;
+		my $ug = new Data::UUID;
+		$prefs->set( server_uuid => lc $ug->create_str() );
 	}
 
 	main::INFOLOG && $log->info("Server binary search path init...");
